build/Settings.props

```diff
@@ -9,7 +9,14 @@
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
     <NoWarn>NU1701;NU5104</NoWarn>
     <IsPackable>false</IsPackable>
-
+    <Authors>OmniSharp contributors</Authors>
+    <PackageTags>omnisharp;lsp;csharp;roslyn;language</PackageTags>
+    <PackageIconUrl>https://avatars0.githubusercontent.com/u/9420663</PackageIconUrl>
```

Comment:
Is embedded icons supported yet?

Reply:
not yet, they merged a basic version recently but it hasn't shipped yet. once it does we can switch https://github.com/NuGet/NuGet.Client/pull/3023